ci/script.sh

```diff
@@ -6,13 +6,16 @@ set -ex
 main() {
     cross build --target $TARGET --release
 
-    if [ ! -z $DISABLE_TESTS ]; then
-        return
+    if [ -z $DISABLE_TESTS ]; then
+        cross test --target $TARGET --features=integration-test --release
     fi
-
-    cross test --target $TARGET --features=integration-test --release
```

Comment:
why changing this part of the script?

Reply:
```
if [ $DISABLE_TESTS is zero length ]; then
   execute tests
fi
```
is the same as 
```
if [ $DISABLE_TESTS is *not* zero length]; then 
    return
fi
Now execute the tests
```
The former is clearer, easier to read and does exactly the same thing as the latter.

Reply:
Let me add a comment to clarify.